spec/integration/custom_error_messages_spec.rb

```diff
@@ -46,5 +46,21 @@
 
       include_context 'schema with customized messages'
     end
+
+    context 'with global configuration' do
+      before do
+        config = Dry::Schema::Config.new
+        config.messages.backend = :i18n
+        allow(Dry::Schema).to receive(:config).and_return(config)
+      end
```

Comment:
We have [a test mode](https://dry-rb.org/gems/dry-configurable/0.11/testing/) for configurables, it would be cleaner and more future-proof if we enabled it.